src/components/DisplayArticles.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import { connect } from 'react-redux';
+
+class DisplayArticles extends Component {
+
+  render() {
+    let articleDisplay = this.props.articles.map(article => {
+      return (
+        <div id={'article-' + article.id} >
```

Comment:
is this supposed to say  ```article-```?

Reply:
Yes, it takes that string and adds the article id, so if it's the first article the id would be article-1...

Reply:
Oh okay .
Thanks for clearing that up for me